pdf2image/pdf2image.py

```diff
@@ -196,6 +207,16 @@ def convert_from_bytes(
             grayscale -> Output grayscale image(s)
     """
 
+    # We make sure that if passed arguments are Path objects, they're converted to strings
+    if isinstance(pdf_file, pathlib.PurePath):
+        pdf_file = pdf_file.as_posix()
+
+    if isinstance(output_folder, pathlib.PurePath):
```

Comment:
I think you can safely remove the checks in `convert_from_bytes` since it ends up using `convert_from_path` which already does the conversion.

Reply:
That is a good point. Somehow these checks being performed twice flew over my head, even though it makes perfect sense after it's been pointed out.
I've removed these in cd83b6c